tests/integration/cypress.config.js

```diff
@@ -77,6 +78,21 @@ module.exports = defineConfig({
       'tests/namespace/test-limit-ranges.spec.js',
       'tests/namespace/test-resource-quotas.spec.js',
       'tests/namespace/z-run-after.spec.js',
+      // uncomment after frog fix
+      // 'tests/kyma-cluster/test-applications.spec.js',
+      // 'tests/kyma-cluster/test-modules-wizard.spec.js',
```

Comment:
Remove this, wizards are no longer in use

Reply:
Removed